WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController/NotificationsViewModel.swift

```diff
@@ -73,4 +95,34 @@ final class NotificationsViewModel {
                 .first(where: notMatcher)
         }
     }
+
+    // MARK: - Handling Inline Actions
+
+    func sharePostActionTapped(with notification: Notification) -> ShareablePost? {
+        guard let url = notification.url else {
+            self.crashLogger.logMessage("Failed to share a notification post due to null url", level: .error)
+            return nil
+        }
+        let content: ShareablePost = (
+            url: url,
+            title: notification.title
+        )
+        self.trackInlineActionTapped(action: .sharePost)
+        return content
+    }
+}
+
+// MARK: - Analytics Tracking
+
+private extension NotificationsViewModel {
+
+    func trackInlineActionTapped(action: InlineAction) {
```

Comment:
Shouldn't it be a private func?

Reply:
AFAIK when an extension is `private`, all methods within are also `private`